//test error

int foo{}

int main()
{
	int x = 0;
	{
		int main() {
      int x = 2;
    }
	}
	return x;
}
